app/controllers/content_management/articles_controller.rb

```diff
@@ -1,11 +1,13 @@
 class ContentManagement::ArticlesController < ApplicationController
 
+    before_action :get_article, only: [:show, :edit, :update]
+
     def index
         @articles = Article.all
     end
 
     def show
```

Comment:
Are you using this route for anything?

Reply:
Yes we have a show view

Reply:
And a before action to get the articles, that's why it seems empty, but it is not ;)